languages/tree-sitter-stack-graphs-javascript/test/base_syntax.js

```diff
@@ -12,6 +12,10 @@ function foo(a) { }
 function foo(undefined) { }
 function* foo() { }
 class Foo { }
+@Foo class Bar { }
+@Foo.Quux class Bar { }
+@Foo() class Bar { }
+@Foo.Quux() class Bar { }
```

Comment:
The latter three lines were all failing with the same error; this PR addresses them all.